components/ui/autocomplete/build.gradle

```diff
@@ -27,6 +27,7 @@ dependencies {
     implementation Dependencies.kotlin_stdlib
 
     implementation Dependencies.support_appcompat
+    implementation project(':concept-toolbar')
```

Comment:
I think we should avoid having this dependency too:

* `ui-*` components are dumb view components without any knowledge about the context they are used in. This ties this to the toolbar.

* The autocomplete component can be used for autocompleting all kinds of things. You could throw it into some app and autocomplete phone numbers or user names. However the concept leaks the idea of "URLs" into it now.

`BrowserToolbar` accepts a an `AutocompleteAcceptor`and in `setAutocompleteListener()` it could create a wrapper/delegate that implements `AutocompleteAcceptor` and delegates to `InlineAutocompleteEditText`. What do you think?

Reply:
I think that's OK. This creates an annoying delegate chain, and I really, really, really dislike delegate chains (you can blame Fennec sync for that...). But it doesn't feel very out of place here, as long as we make sure this doesn't set a wider precedent for these damn things ;)